### DISCUSSION TITLE
2024-02-17-力扣第124场双周赛 | 阿聪的小破站

### DISCUSSION THREAD

**Discussion by giscus** [SmartDengC/jet5devil-index#9] [Q&A]
# 2024-02-17-力扣第124场双周赛 | 阿聪的小破站

https://wiki.dengcong.com.cn/article/9l5yd941/

**Comment by SmartDengC** [comment]
现在这个阶段，主要弄懂周赛的前面两道题就可以，因为我现在只能做对一道 （😂）

**Reply by SmartDengC** [reply]
为啥一敲回车就给我comment了，如何实现换行？
在reply里面enter直接就是换行

**Reply by SmartDengC** [reply]
解密了，直接enter的话是换行，如果是ctrl+enter就是commenct，上面写错了！

**Comment by SmartDengC** [comment]
补充说明第二道题：
1、sorted(last[ch] for ch, c in cnt.items() if c == mx)  为什么这里要用sorted来排一下序？

**Reply by SmartDengC** [reply]
因为这里求出来的是元素的下标，最后需要从下标小的到大的进行输出
